gnomad_qc/v4/annotations/insilico_predictors.py

```diff
@@ -89,6 +90,131 @@ def _load_cadd_raw(cadd_tsv) -> hl.Table:
     return ht
 
 
+def create_spliceai_grch38_ht(
+    snvs_path: str, indels_path: str, new_indels_path: str, header_file_path: str
```

Comment:
I think we should be consistent with how we handle paths in the same module. CADD's import has all paths inside the function. Lets move spliceAIs paths into the function as well and remove the parameters so in main youd just call `create_spliceai_grch38_ht()`